spread.yaml

```diff
@@ -796,6 +796,26 @@ prepare: |
         apt-get update && apt-get install -y eatmydata
     fi
 
+    if [[ "$SPREAD_SYSTEM" == ubuntu-fips-* ]]; then
+        pro status
+
+        if [[ "$SPREAD_REBOOT" = 0 ]]; then
+            echo "--- enabling FIPS"
+            pro enable fips-updates --assume-yes
+            REBOOT
+        else
```

Comment:
I guess that since REBOOT returns, you can just drop the else part remove the indent level as it will be all the same.